README.md

```diff
@@ -9,11 +9,11 @@ The tool can be used also as a simple MCDA ranking tool with no variability (see
 
 ### Input information needed in the configuration file
 The following input information are contained in the `configuration.json` file:
-- input matrix, a table where rows represent the alternatives and the columns represent the indicators. Be sure that there is no index column but the column with the alternative names is set as index by:
+- input matrix, a table where rows represent the alternatives and the columns represent the indicators. Be sure that there is not an index column but that the column with the alternative names is set as index by:
 ```bash
-input_matrix = input_matrix.set_index('Alternatives'),
+input_matrix = input_matrix.set_index('Alternatives').
 ```
-and that there are no duplicates among the rows.
+Be also sure that there are no duplicates among the rows. If the values of one or more indicators are all the same, the indicators are dropped from the input matrix because they do not carry any information.
```

Comment:
> If the values of one or more indicators are all the same

Is it about the duplicate values or the missing values? 
From the code `DataFrame.nunique` it appears that there can be a single value that is duplicated along with some NaNs.

From the PR description it appeared that you wanted to drop off columns with missing and a single value.

Reply:
I never mentioned missing values, have I? This might be also something to think about.

Reply:
If everything is clarified, I'll merge.